teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far! It would be nice if you add `age, specialization, name = "Unknown", parent_permission = true`  to youe `def` 🙏🏼 

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @specialization = classroom
  end
```